sql/step_XX_drop_adm_deleted.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE patient DROP COLUMN pat_deleted;
```

Comment:
Note the file name: `step_XX_`  and the contents of this file is the same as the next: `ALTER TABLE patient ...`